cmd/webhook/admission/admission_test.go

```diff
@@ -133,6 +133,11 @@ func TestRouteGroupAdmitter(t *testing.T) {
 			inputFile: "rg-with-multiple-predicates.json",
 			message:   `single predicate expected at \"Method(\\\"GET\\\") && Path(\\\"/\\\")\"\nsingle predicate expected at \" \"`,
 		},
+		{
+			name:      "routegroup with invalid backends",
+			inputFile: "rg-with-invalid-backend-path.json",
+			message:   `backend address \"http://example.com/foo\" contains path or query\nbackend address \"http://example.com/foo/bar\" contains path or query\nbackend address \"http://example.com/foo/\" contains path or query\nbackend address \"/foo\" contains path or query\nbackend address \"http://example.com/\" contains path or query\nbackend address \"example.com/\" contains path or query\nbackend address \"example.com/foo\" contains path or query\nbackend address \"http://example.com?foo=bar\" contains path or query`,
+		},
```

Comment:
Do we need an essentially copy-pasted test case for the webhook given that it relies on the dataclients Validator that has its own test?

Reply:
If both validators have the same options then no, but maybe webhook has more strict way of validation later. Also this tests webhook response, so the error maybe the same but the response is different.